src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -10457,6 +10979,19 @@ namespace HVACVariableRefrigerantFlow {
             this->FanPower = Fans::GetFanPower(this->FanIndex);
         }
 
+        // run supplemental heating coil
+        Real64 SuppPLR = this->SuppHeatPartLoadRatio;
```

Comment:
Same comment, if SuppPLR is not used elsewhere it can be moved inside the IF block to execute only if a supp coil exists.

Reply:
@rraustad agree. Done.